web/bindata.go

```diff
@@ -112,7 +112,7 @@ func indexHtml() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "index.html", size: 2615, mode: os.FileMode(420), modTime: time.Unix(1491867544, 0)}
```

Comment:
I am not sure how this line was removed. I'll have to do some digging to see how it got removed.